src/components/canvas-tools/templates/Line.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div>
+    <v-color-picker
+      v-model="lineColour"
+      :swatches="swatches"
+      mode="hexa"
+      show-swatches
+      hide-canvas
+      flat
+      hide-inputs
+      hide-mode-switch
+    />
+    <v-slider
+      prepend-icon="fa-ruler-vertical"
+      :hint="lineSizeHint"
+      v-model="lineSize"
+      :max="6"
+      :step="1"
+      ticks="always"
+    />
+    <v-radio-group v-model="lineEndStyle" row>
+      <v-spacer></v-spacer>
+      <v-radio class="mr-0" name="endStyle" on-icon="fa-arrow-right" off-icon="fa-arrow-right" :value="'arrow'"></v-radio>
+      <v-spacer></v-spacer>
+      <v-radio class="mr-0" name="endStyle" on-icon="fa-slash" off-icon="fa-slash" :value="'line'"></v-radio>
+      <v-spacer></v-spacer>
+      <v-radio class="mr-0" name="endStyle" on-icon="fa-ruler-combined" off-icon="fa-ruler-combined" :value="'T'"></v-radio>
+      <v-spacer></v-spacer>
+    </v-radio-group>
+    <v-radio-group v-model="lineStrokeStyle" row>
+      <v-spacer></v-spacer>
+      <v-radio class="mr-0" name="endStyle" on-icon="fa-minus" off-icon="fa-minus" :value="0"></v-radio>
+      <v-spacer></v-spacer>
+      <v-radio class="mr-0" name="endStyle" on-icon="fa-ellipsis-h" off-icon="fa-ellipsis-h" :value="1"></v-radio>
+      <v-spacer></v-spacer>
+    </v-radio-group>
+    <v-container>
+      <v-row>
+        <v-spacer></v-spacer>
+        <v-switch
+          v-model="lineTemporarily"
+          :label="lineTemporarily === true ? 'Temporarily' : 'Permanent'"
```

Comment:
what keyword are we going for here, Temporary or Temporarily? Personally I prefer the first one?
If lineTemporarily is a boolean then you don't need the `=== true`.
`lineTemporary ? 'Temporary' : 'Permanent'` will work

Reply:
Going for 'Temporary'